spec/board_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../lib/board.rb'
+
+describe Board do
+  describe '#integer_count' do
+    it 'returns true if return value is an integer or nil' do
+      board = Board.new
+      count = board.integer_count
+      expect(count).to be_a Integer || Nil
+    end
```

Comment:
Let's test for both positive and negative cases. 
Don't test two cases in on `expect` block, by two cases I meant `integer or nil`, test for when it will return an `integer` and when it will return `nil` separately, but in the same `describe '#integer_count'` block.


Also let's make the test description more aligned with what we testing, instead of `returns true if return value is an integer or nil` why not use `returns an integer when there are values in the board`, this way we know a bit about what `integer_count` is used for.



> A bit of common sense: check if specs cover positive and negative scenarios ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)) Example: there is a spec for correct usage of method \[e.g. `it "creates object if all required attributes are passed"`\] and also there is spec for wrong usage of method \[e.g. `it "does not create object if required attributes are missing"`\].